tests/unit/App.spec.ts

```diff
@@ -17,147 +17,147 @@ const vuetify = new Vuetify({})
 const store = getVuexStore()
 
 const BCOMP_ADDRESSES = {
-  'registeredOffice': {
-    'mailingAddress': {
-      'streetAddress': '1012 Douglas St',
-      'addressCity': 'Victoria',
-      'addressRegion': 'BC',
-      'addressType': 'mailing',
-      'postalCode': 'V8W 2C3',
-      'addressCountry': 'CA'
+  registeredOffice: {
```

Comment:
I thought I had some test errors here and cleaned up along the way... As it turns out, there were no changes required here but I thought I'd keep the cleanup instead of throw it away.